### PR TITLE
Harmonise wording for techniques checks/steps

### PR DESCRIPTION
**Pull request by patrickhlauke**
Currently, techniques use a mix of wording `If step X...` and `If check X...`, as well as a few outlier "If test procedure X...". Most prefix the step/check number with `#`, but some don't. And conceptually, I think it makes more sense saying that a *check* is true/false, rather than a *step* being true/false.

This PR tries to harmonise this, settling for `check` and adding missing `#` (though, it has to be said, this use of the octothorpe is a bit US-centric, so maybe we want to harmonise to *not* having it?)

### CODE REVIEW

**Comment by fstrr** [comment]
Agree on the US-centric use of the `#` symbol. I've often considered yoinking it out and would be fine with it being removed.

**Review by fstrr — APPROVED** [review]

**Comment by patrickhlauke** [comment]
Point to discuss in our next WCAG 2.x meeting then ... if we want to yoink `#` out, I can do a big find/replace and add it as a commit here.

**Comment by alastc** [comment]
I think aligning with the earlier examples is probably best (and the least amount of files changed), which is what you've done. That will also ruffle the least feathers.

File: techniques/failures/F52.html
Comment (mgifford) [inline techniques/failures/F52.html:1]: I think this is better as check #2 and check #3 but in other places it is checks #2 and #3. Just wanted to highlight this.

File: techniques/failures/F9.html
Comment (mgifford) [inline techniques/failures/F9.html:1]: This is inconsistent. Should be all checks and no steps, right?

**Review by ljoakley — APPROVED** [review]
completed with a few changes

File: techniques/failures/F24.html [hunk 1]
```diff
@@ -90,7 +90,7 @@ <h3>Specifying foreground color of link text with CSS</h3>
          </div>
       </section>
       <section class="results"><h3>Expected Results</h3>
-         <p>If step #2 is true but step #3 is false, OR if step #3 is true but step #2 is false then this
+         <p>If check #2 is true but step #3 is false, OR If check #3 is true but step #2 is false then this
```
Comment (patrickhlauke) [inline techniques/failures/F24.html:93]:
```suggestion
         <p>If check #2 is true but step #3 is false, <em>or</em> if check #3 is true but step #2 is false then this
```

File: techniques/failures/F38.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
       </section>
       <section class="results"><h3>Expected Results</h3>
          <ul>
-            <li>If step #1 is true and if step #2 is true, this failure condition applies and content fails the Success Criterion.</li>
+            <li>If check #1 and #2 is true, this failure condition applies and content fails the Success Criterion.</li>
```
Comment (patrickhlauke) [inline techniques/failures/F38.html:25]:
```suggestion
            <li>If check #1 and #2 are true, this failure condition applies and content fails the Success Criterion.</li>
```

**Comment by mbgower** [comment]
@shawna-slh Is there any style decision on use of the pound symbol (#) in front of ordered list references?

**Comment by shawna-slh** [comment]
Thanks for the ping @mbgower ! We don't currently have that in our Style guide. (I prefer without the # - yet don't have any support for or against that.) From a quick search, I don't find guidance in leading Style Guides -- so it needs more research.

The good news is we have a new WAI staff person coming on the end of the month who will be great at citing sources for a decision.

Are y'all comfortable leaving this issue open for a few weeks until we can get back to it?

(p.s. thanks to @patrickhlauke I learned a new term, "octothorpe".)

**Comment by patrickhlauke** [comment]
> Are y'all comfortable leaving this issue open for a few weeks until we can get back to it?

no rush from my perspective. I'd rather wait and then do all the changes in a oner, once we have a decision on the `#`

**Comment by mbgower** [comment]
@shawna-slh Sorry, I missed those final comments when I brought this into our current review cycle.
If someone gets to this by the start of August, we can just incorporate your change, otherwise we'll just push a second update when you make your recommendation.

**Comment by shawna-slh** [comment]
@tamsinewing555 will probably get to it within a few days :)

**Pull request merged**